examples/text_classification-ort.ipynb

```diff
@@ -0,0 +1,1289 @@
+      "2022-09-16 13:13:24.566274859 [W:onnxruntime:, graph.cc:3472 CleanUnusedInitializersAndNodeArgs] Removing initializer 'org.pytorch.aten::ATen_119'. It is not used by any node and should be removed from the model.\n",
+      "2022-09-16 13:13:24.566278842 [W:onnxruntime:, graph.cc:3472 CleanUnusedInitializersAndNodeArgs] Removing initializer 'org.pytorch.aten::ATen_118'. It is not used by any node and should be removed from the model.\n",
+      "2022-09-16 13:13:24.566289045 [W:onnxruntime:, graph.cc:3472 CleanUnusedInitializersAndNodeArgs] Removing initializer 'org.pytorch.aten::ATen_122'. It is not used by any node and should be removed from the model.\n",
+      "2022-09-16 13:13:24.566329639 [W:onnxruntime:, graph.cc:3472 CleanUnusedInitializersAndNodeArgs] Removing initializer 'org.pytorch.aten::ATen_124'. It is not used by any node and should be removed from the model.\n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "\n",
+       "    <div>\n",
+       "      \n",
+       "      <progress value='66' max='66' style='width:300px; height:20px; vertical-align: middle;'></progress>\n",
+       "      [66/66 00:00]\n",
+       "    </div>\n",
+       "    "
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "{'eval_loss': 0.8077986240386963,\n",
+       " 'eval_matthews_correlation': 0.5327637463001902,\n",
+       " 'eval_runtime': 5.997,\n",
+       " 'eval_samples_per_second': 173.92,\n",
+       " 'eval_steps_per_second': 11.005,\n",
+       " 'epoch': 5.0}"
+      ]
+     },
+     "execution_count": 41,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "trainer.evaluate(inference_with_ort=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a32ba2ab-ccc3-4669-b04c-a22bdf08ecd8",
+   "metadata": {},
+   "source": [
+    "## __Extended reading__\n",
+    "\n",
+    "Now check your trained ONNX model with Netron, and you might notice that the computation graph is under optimizatiom. Want to accelerate even more? \n",
+    "\n",
+    "Check the graph optimizers and quantizers of [Optimum](https://github.com/huggingface/optimum)🤗! "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4d2ee6ee-6b52-4e40-a819-872d308a31c1",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.10"
+  }
```

Comment:
I would rename the file `text_classification_ort.ipynb` to keep consistency with the existing ort examples